ovos_bus_client/util/scheduler.py

```
@@ -30,9 +30,8 @@
 
 from ovos_config.config import Configuration
 from ovos_config.locations import get_xdg_data_save_path, get_xdg_config_save_path
-from ovos_utils.log import LOG, log_deprecation
-from ovos_utils.events import EventContainer as _EventContainer
-from ovos_utils.events import EventSchedulerInterface as _SchedulerInterface
+from ovos_utils.log import LOG
+from ovos_utils.events import EventContainer, EventSchedulerInterface
```

Ruff

`ovos_utils.events.EventContainer` imported but unused

Remove unused import

(F401)